.rtd-environment.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - numpy=1.14
   - scipy=1.0 # needed for coordinates cross-matching
   - jupyter=1.0
-  - notebook=5.4
+  - notebook>=5.7.2
```

Comment:
```suggestion
  - notebook>=5.7.2
  - sphinx-astropy
```

Reply:
p.s. Well, the failure is at CircleCI, so probably need to update the other files as well but you get the idea.

p.p.s. Albeit this being an educated guess, I could be wrong. Only one way to find out!

Reply:
This is RTD - we actually don't use RTD anymore, and sphinx-astropy is in the circleci config

Reply:
Well... Ignore me then... :sweat_smile: